view/theme/frio/css/style.css

```diff
@@ -2045,14 +2073,19 @@ ul.tabbar ul.tabs-extended li.active a {
     background: none;
 }
 ul.dropdown-menu li:hover {
-    border-bottom-width: 0;
+    border-bottom: 0;
+}
+.dropdown-menu li {
+    display: block;
+    width: 100%;
+    box-sizing: border-box;
 }
-
-
 /* Dropdown Menu */
 .dropdown-menu li a,
 .dropdown-menu li .btn-link {
-	color: $link_color;
+    /* color: $link_color; */
```

Comment:
Please remove commented out properties. The git history will allow us to find previous code.

Reply:
Yeah that was for testing, again. I hope I won't always leave comments behind :joy:

Reply:
Don’t worry, I’ll point them out every single time! 😁